### ISSUE TITLE
[Feature Request] Installation dialogue for games installed from pkgdir

### ISSUE THREAD

**Issue by Darkhost1999** [RPCS3/rpcs3#12177]
## Quick summary
Some discs have pkgs on the disc inside PS3_GAME/PKGDIR

## Details
The current behavior is RPCS3 just says (not responding) during first boot of Bioshock infinite while installing hdd game Bioshock (included on disc) and during first boot of Assassin's Creed Revelations installing hdd game Assassin's Creed (included on disc).

There is no user feedback that anything is happening at all so we're looking at the screen like what is happening?
For those knowledgeable by having been informed about the issue previously know that all that is happening is you're installing a hdd game from your disc so give it few minutes.

We should probably include an option to not install the hdd game upon users request and they can tick "don't show this message again" for those who just don't want to install the game.
If they did want to install it they can always manually drag the PKG onto RPCS3's UI to install the pkg.

Current behavior forces users to install the hdd game without permission and sometimes without the user's knowledge.

#### 1. Please describe, what part of RPCS3 would be affected by your feature:
* UI
* Game Installation

#### 2. Please tell us, why your feature is important to RPCS3.
To inform users that RPCS3 is working as intended.

#### 3. Please attach screenshots of the feature implemented in other projects.

I'm not aware of any.
We just need a qmessagebox or something to reflect information to the user.

#### 4. Please provide your system configuration:
* Win11
* R5 3600
* GTX 1660S

#### Please include.
* Anything else you deem to be important

If you want to throw in detecting and installing dlc pkgs from the subdirectories of the PS3_EXTRA folder of games like skyrim to install DLC from disc at first boot like how is already implemented for hdd games installing from pkgdir it wouldn't hurt anything.

**Comment by Darkhost1999** [comment]
Installing the pkgdir pkg before booting the game that has a pkgdir folder still says not responding and overwrites the already installed data instead of detecting that it is already installed. (Aka manually trying to workaround the issue doesn't work)

Real ps3 behavior will show both icons on xmb. And when booting the disc game will boot the disc game without even trying to install the pkg. Allowing the user to install the hdd game at their own leisure. But always providing the option to install if the user wants to.

**Comment by Darkhost1999** [comment]
I've been putting thought into how I would do this.
I would just cut off installing the pkgs from pkgdir at disc boot because the ps3 doesn't do that.
Then I would
IF hdd game = installed, display game from dev_hdd0/game not pkgdir.
IF game = not installed add the game from pkgdir to the game's list.
I believe that's possible from the following images. We could keep it simple and just check for games serial.

<details>
<summary>Images of file explorer</summary>

![4.png](https://user-images.githubusercontent.com/60384196/182026466-034e2776-02d1-4b41-8db9-b4b2a9333854.png)
![3.png](https://user-images.githubusercontent.com/60384196/182026470-ddf7310a-7a5e-4dcd-a36e-cc07a97a6ccc.png)
![2.png](https://user-images.githubusercontent.com/60384196/182026703-03d9385d-4fe7-4519-946c-1e403b9d8626.png)
![1.png](https://user-images.githubusercontent.com/60384196/182026706-274c0648-b459-4f56-8a3b-4c47bbefd12b.png)

</details>

When they go to launch the uninstalled game it could say "would you like to install <gamename>.pkg" Then it should just utilize the existing pkg installer information as feedback to the user.

**Comment by Darkhost1999** [comment]
If there's multiple pkgs on a disc the directory names will just count by 1.
PKG00
PKG01
PKG02
And so on.

**Comment by Darkhost1999** [comment]
I noticed this
<https://github.com/RPCS3/rpcs3/pull/3799>
And am curious if the https://github.com/RPCS3/rpcs3/blob/master/rpcs3/rpcs3qt/main_window.h#L152 HandlePackageInstallation function can be used in place of the https://github.com/RPCS3/rpcs3/blob/master/rpcs3/Emu/System.cpp#L1655 rpcs3::utils::install_pkg function.

So I decided to try and give it a try just in case on my local pc to look at. I got an #include <QWinThumbnailToolBar> file or directory could not be found error when compiling. So I sent it to Cirrus. To get the same errors. And I have even changed the includes locally to as they are written on the qt docs instead of QWinTHumbnailtoolBar. But none of the errors changed at all. I'm sure there's more work that actually needs to be done maybe. Yet I was completely unable to test my theory because of this problem. I just want the existing pkg installer to be called and ask the user if they want to install the pkg or not. Then using the existing progress bars show how long it will take to install the files without just showing RPCS3 as not responding.
